Apps/ClinicalDocument/src/Services/ClinicalDocumentService.cs

```diff
@@ -106,7 +113,24 @@ public async Task<RequestResult<EncodedMedia>> GetFileAsync(string hdid, string
             RequestResult<PersonalAccount?> response = await this.personalAccountsService.GetPatientAccountAsync(hdid).ConfigureAwait(true);
             if (response.ResultStatus == ResultType.Success)
             {
+                string? pid = response.ResourcePayload?.PatientIdentity?.Pid.ToString();
                 this.logger.LogDebug("PID Fetched: {Pid}", response.ResourcePayload?.PatientIdentity?.Pid);
```

Comment:
Nit: you can reference the variable that was just defined to make the log statement a tad cleaner.

Reply:
Yeah looks like I started to do that then lost it. I'll fix that up with my unit tests.